src/docs/manual/03_task_exportDrawIo.adoc

```diff
@@ -1,6 +1,6 @@
 :filename: manual/03_task_exportDrawIo.adoc
 ifndef::imagesdir[:imagesdir: ../images]
-:experimental:
```

Comment:
:experimental: gives you access to the Menu and keyboard macros which I've used on line 18. Without that macro, the keyboard macro won't render correctly.

Reply:
ah. ok. didn't see that. let's put it back in...

Reply:
ah - I've seen you've already put it back in. perfect